pex/environment.py

```diff
@@ -348,12 +349,23 @@ def _resolve_requirement(
         resolved_dists_by_key,  # type: MutableMapping[_RequirementKey, FingerprintedDistribution]
         required,  # type: bool
         required_by=None,  # type: Optional[Distribution]
+        exclude_configuration=ExcludeConfiguration(),  # type: ExcludeConfiguration
```

Comment:
Minor thing, but I notice that both callers of this private function currently provide this argument. Could it potentially be a required argument, so that future refactoring doesn't accidentally leave it off when it should be provided?

Reply:
Sure. I did not like the re-ordering this forced, which mixes topic importance to my reading, but done.